schemas/lightning-preview-app.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
I removed this file since for now `LightningPreviewApp` extends from `SfCommand<void>` and does not return anything. As we work on adding more functionalities to `LightningPreviewApp`, we can change the return type as needed and auto-generate the new schema